src/EventStore.Core/ClusterVNode.cs

```diff
@@ -289,21 +287,21 @@ public class ClusterVNode<TStreamId> :
 
 			var extTcp = disableExternalTcpTls && enableExternalTcp
 				? new IPEndPoint(options.Interface.NodeIp,
-					extTcpOptions.Port)
+					extTcpOptions.ExtPort)
```

Comment:
for the sake of making it easier to migrate to the plugin it might be nice to keep the old names `NodeTcpPort` `NodeTcpPortAdvertiseAs`

Reply:
My logic was we deprecated those names I used but those were never actually removed. So I thought keeping the old names was the least likely to break.